packages/client/hmi-client/src/services/charts.ts

```diff
@@ -577,3 +594,30 @@ export const createOptimizeForecastChart = (
 
 	return spec;
 };
+
+export const createInterventionChart = (interventionsData: { name: string; value: number; time: number }[]) => {
+	const spec: any = {
+		$schema: VEGALITE_SCHEMA,
+		width: 400,
+		autosize: {
+			type: 'fit'
+		},
+		layer: []
+	};
+	if (interventionsData && interventionsData.length > 0) {
+		// markers
+		createInterventionChartMarkers(interventionsData).forEach((marker) => {
+			spec.layer.push(marker);
+		});
+		// chart
+		spec.layer.push({
+			data: { values: interventionsData }, // Dummy data to ensure the layer is rendered
```

Comment:
What is up with the comment? 
If we have a problem with Vegalite I would rather have a proper solution rather than a hack-fix.

Reply:
Sorry it wasnt meant to be there, copy pasted it from the markers above.  I fixed those markers anyways with the latest commit and removed the comment